datacats/error.py

```diff
@@ -25,3 +25,36 @@ def pretty_print(self):
         for line in self.message.format(*self.format_args).split('\n'):
             print "  ", line
         print colored.blue("-" * 40)
+
+
+class WebCommandError(Exception):
+
+    def __init__(self, command, container_id, logs):
+        self.command = command
+        self.container_id = container_id
+        self.logs = logs
+
+    def __str__(self):
+        return \
+            ('\nDocker container "/web" command failed\n'
+             '    Command: {0}\n'
+             '    Docker Error Log:\n'
+             '    {1}\n'
+             ).format(" ".join(self.command), self.logs, self.container_id)
+
+
+class RemoteCommandError(WebCommandError):
+    def __init__(self, base_WebCommandError):
+        self.__dict__ = base_WebCommandError.__dict__
```

Comment:
"where in the heck did my RemoteCommandError.logs value come from" is harder to understand with this code. Also, it's just weird to have to pass an exception to another exception's __init__.

Does the __dict__ copying prevent losing the traceback from the original exception?

Thinking about this more, why don't we pass the exception class to use to web_command (defaulting to WebCommandError) instead?